NetKVM/Common/ParaNdis-SM.h

```diff
@@ -2,18 +2,21 @@
 
 #include "ParaNdis-Util.h"
 
-class CDataFlowStateMachine : public CPlacementAllocatable
+class CFlowStateMachine : public CPlacementAllocatable
 {
 public:
-    void Start()
+    virtual void Start()
```

Comment:
I usually don't argue against virtual methods if they make the code more elegant and readable. But in this case I don't follow why you made all of these virtual. They don't seem to be overriden anywhere. Is it in preparation for further changes? Thanks!

Reply:
Well no there is no upcoming change soon, but I though it would be more elegant this way as it emphasis that this methods can be overridden.

If you think this isn't good this can be reverted by a suitable patch =)

Reply:
I'm fine with it either way :) Thanks!